api/http/dataplane/dataplane-api.yaml

```diff
@@ -145,26 +145,38 @@ components:
     ConfigurationStatus:
       type: object
       properties:
+        instanceID:
```

Comment:
Think we are using `Id` in the REST API field names instead of `ID`

Reply:
Will fix, but is there a reason we are using InstanceId here but InstanceID everywhere else ?